client/scripts/services/db.js

```diff
@@ -136,7 +137,24 @@ require('../app').service('db', /* @ngInject */function ($q, Location, Nomenclat
       return res
     }).finally(function () {
       delete db.visits.$promise
-    })
+    });
+
+  (db.$updateOrganizations = function () {
+    db.organizations = {}
```

Comment:
This initialization is better outside of the method, so on consecutive updates the list is not cleared